packages/account/src/Components/poi/poi-confirm-with-example-form-container/poi-confirm-with-example-form-container.tsx

```diff
@@ -81,12 +84,27 @@ const PoiConfirmWithExampleFormContainer = ({
         );
         const data = await WS.setSettings(request);
 
-        if (data.error) {
-            setStatus({ error_msg: data.error.message });
+        const generic_error_message = (
+            <Localize i18n_default_text='Sorry, an internal error occurred. Hit the above checkbox to try again.' />
+        );
+
+        const duplicated_account_error_message = (
+            <Localize
+                i18n_default_text='An account with these details already exists. Please make sure the details you entered are corrrect as only one real account is allowed per client. If this is a mistake, contact us via <0>live chat</0>.'
+                components={[
+                    <span key={0} className='link link--orange' onClick={() => window.LC_API.open_chat_window()} />,
+                ]}
+            />
+        );
+
+        if (data?.error) {
+            const response_error =
+                data.error?.code === 'DuplicateAccount' ? duplicated_account_error_message : generic_error_message;
+            setStatus({ error_msg: response_error });
             setSubmitting(false);
         } else {
             const response = await WS.authorized.storage.getSettings();
```

Comment:
This also `useSettings`. We can also invalidate if you need to refetch. Plus here we should be expecting recent data instead of cached one